docs/current/sdk/go/guides/110632-embed-directories.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: /110632/embed-directories
+---
+
+# Copy embedded directories into a container
+
+Dagger does not expose the option to copy entire directories as a single step (yet). It is, however, doable by walking over it.
```

Comment:
Is there any issue that track this feature? This is mandatory to use Dagger in local project.

Reply:
It's not related to local directories, but the creation of a directory recursively. It works when copying/mounting from host. But cp a dir recursively is not implemented yet